core/Updates/2.15.0-b1.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Option;
+use Piwik\Plugins\SitesManager\API;
+use Piwik\Plugins\SitesManager\Model;
+use Piwik\Updater;
+use Piwik\Updates;
+
+class Updates_2_15_0_b1 extends Updates
+{
+    public function doUpdate(Updater $updater)
+    {
+        $this->reformatWebsiteSettings();
+    }
+
+    /**
+     * Some website settings are now separated by line returns instead of commas
+     */
+    private function reformatWebsiteSettings()
+    {
+        // Excluded query parameters
+        $globalExcludedQueryParameters = Option::get(API::OPTION_EXCLUDED_QUERY_PARAMETERS_GLOBAL);
+        $globalExcludedQueryParameters = str_replace(',', "\n", $globalExcludedQueryParameters);
+        Option::set(API::OPTION_EXCLUDED_QUERY_PARAMETERS_GLOBAL, $globalExcludedQueryParameters);
```

Comment:
FYI: We should write raw SQL statements here instead of using `Option::set`. If we ever change something with the `option` table or `Option` API this update might fail as it will have an "old" structure